freqtrade/exchange/exchange.py

```diff
@@ -1894,6 +1894,8 @@ def _set_leverage(
         except (ccxt.NetworkError, ccxt.ExchangeError) as e:
             raise TemporaryError(
                 f'Could not set leverage due to {e.__class__.__name__}. Message: {e}') from e
+        except ccxt.BadRequest:
+            return
```

Comment:
This error is way to broad - i think there's a multitude of reasons there could be a "BadRequest".
"hiding" these in general can hide problems we'd encounter for other exchanges.

Reply:
How can I properly test this method with my api keys so that I can see the structure of the error and make it more specific

Reply:
Run freqtrade with your API keys in debug mode? 
i mean - it's a simple configuration - and then you just input the regular command as "debug command" in vscode ...